src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -10,6 +10,7 @@
 
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
+@CrossOrigin(origins =  "https://glacial-reef-77706.herokuapp.com")
```

Comment:
Czy to jest adres, na którym stoi frontend?

Reply:
@interceptor2048 Tak